### PR TITLE
[Dev] Removing broken link, changing DateTimePicker to RNDateTimePick…

### PR DESCRIPTION
**Pull request by jefelewis**
…er to be more uniform with the rest of the docs, and formatting

# Summary

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [ ] I have tested this on a device and a simulator
- [X] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [ ] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by jefelewis** [comment]
Also, if I import the DatePicker with having the SafeAreaView with `<SafeAreaView style={{ display: 'flex', flex: 1 }}>`, I get a different error: Invariant Violation: requireComponent: "RNDateTimePicker" was not found in the UIManager"

```javascript
const App = () => {
  return (
      **<SafeAreaView style={{ display: 'flex', flex: 1 }}>**
        <DatePicker
          title="Date"
          onChange={(date) => console.log(date)}
          mode="spinner"
        />
      </SafeAreaView>
  );
};
```

Simulator Screen Shot - iPhone 11 - 2020-01-22 at 17 26 17

File: README.md [hunk 1]
```diff
@@ -23,7 +23,6 @@ React Native date & time picker component for iOS and Android
       - [iOS](#ios)
       - [Android](#android)
   - [Getting started](#getting-started)
-    - [Install using react-native link](#install-using-react-native-link)
```
Comment (jefelewis) [inline README.md]: Broken link

File: README.md [hunk 2]
```diff
@@ -147,21 +146,21 @@ yarn add @react-native-community/datetimepicker
 ## General Usage
 
 ```js
-import DateTimePicker from '@react-native-community/datetimepicker';
```
Comment (jefelewis) [inline README.md]: Not consistent with the rest of the docs where it is used as `RNDatetimePicker`

**Comment by vonovak** [comment]
hello, thanks for the PR. I'll fix the broken link, and with regard to RNDateTimePicker vs DateTimePicker - referring to the module by `DateTimePicker` from JS is preferable as the RN prefix doesn't really bring value and the RN prefix in JS is not used in other RN packages either. The reason for the prefix is objective c where we need to the classes to have unique names across the project, hence you'll see RCT (=react), RN (=react native) or RNC (=react native community) being used.

**Pull request closed without merging**